src/server/lib/cli.ml

```diff
@@ -0,0 +1,52 @@
+(*
+  This file is part of scilla.
+  Copyright (c) 2018 - present Zilliqa Research Pvt. Ltd.
+  scilla is free software: you can redistribute it and/or modify it under the
+  terms of the GNU General Public License as published by the Free Software
+  Foundation, either version 3 of the License, or (at your option) any later
+  version.
+  scilla is distributed in the hope that it will be useful, but WITHOUT ANY
+  WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+  A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+  You should have received a copy of the GNU General Public License along with
+  scilla.  If not, see <http://www.gnu.org/licenses/>.
+*)
+
+open Core
+open Api
+open ScillaUtil.FilePathInfix
+
+module Cmds = API(Cmdlinergen.Gen ())
+
+module Cmd = struct
+  open Cmdliner.Term
+
+  let default ~version =
+    let doc = "The Scilla server CLI" in
+    ret (const (fun _ -> `Help (`Pager, None)) $ const ()),
+    info "scilla-server" ~version ~doc
+
+  let server ~sock_path ~num_pending =
+    const @@ Server.start ~sock_path ~num_pending $ const (),
+    info "start" ~doc:"Start Scilla server"
+end
+
+let run ~sock_path ~num_pending =
+  let open Cmdliner in
+  let logs_dir = "_build" ^/ "logs" in
+  Util.mkdir_rec ~dir:logs_dir ~perm:0o0755;
+  GlobalConfig.(
+    set_use_json_errors true;
+    set_log_file @@ logs_dir ^/ "scilla-server.log";
+    set_debug_level Debug_Normal
```

Comment:
Let's have `Debug_none` as the default. I suspect that it's safer in terms of performance.